module.exports = {
    "0": "No errors occurred, and no copying was done. The source and destination directory trees are completely synchronized.",
    "1": "One or more files were copied successfully (that is, new files have arrived).",
    "2": "Some Extra files or directories were detected. No files were copied. Examine the output log for details.", 
    "4": "Some Mismatched files or directories were detected. Examine the output log. Some housekeeping may be needed.",
    "8": "Some files or directories could not be copied (copy errors occurred and the retry limit was exceeded). Check these errors further.",
    "16": "Serious error. Robocopy did not copy any files. Either a usage error or an error due to insufficient access privileges on the source or destination directories.",
    
    "3": "(2+1) Some files were copied. Additional files were present. No failure was encountered.",
    
    "5": "(4+1) Some files were copied. Some files were mismatched. No failure was encountered.",
    "6": "(4+2) Additional files and mismatched files exist. No files were copied and no failures were encountered. This means that the files already exist in the destination directory",
    "7": "(4+1+2) Files were copied, a file mismatch was present, and additional files were present."
};
